code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -362,19 +362,21 @@ func (ssc *StorageSmartContract) GetReadMarkersHandler(ctx context.Context,
 	resp interface{}, err error) {
 
 	var (
-		allocationID = params.Get("allocation_id")
-		authTicket   = params.Get("auth_ticket")
+		allocationID       = params.Get("allocation_id")
+		authTicket         = params.Get("auth_ticket")
+		offsetString       = params.Get("offset")
+		limitString        = params.Get("limit")
+		isDescendingString = params.Get("is_descending")
```

Comment:
keep a key, sort:-1/1
-1 can be descending and 1 can be ascendind

please do similar for writemarker as well

Reply:
I don't feel -1 and 1 are intuitive ... maybe we can find a better way?

Reply:
you can use sort=asc/desc

Reply:
done